packages/react-core/src/components/DataList/__tests__/DataList.test.tsx

```diff
@@ -163,7 +162,7 @@ describe('DataList', () => {
   test('DataListAction visibility - hide button on 2xl', () => {
     const view = shallow(
       <DataListAction
-        className={css(DataListActionVisibility.hiddenOn2Xl)}
+        visibility={{ default: 'hidden', '2xl': 'hidden' }}
```

Comment:
the default should be visible here, no?

Reply:
Yeah, the `default` shouldn't be there. Will update in next commit that also bumps core.